src/store/layout/index.ts

```diff
@@ -1,19 +1,16 @@
-import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction, createAction } from '@reduxjs/toolkit';
+import type { AppLayout, LayoutState, UpdateSidekickPayload } from './types';
 
-export interface AppLayout {
-  hasContextPanel: boolean;
-  isContextPanelOpen: boolean;
-  isSidekickOpen: boolean;
+export enum SagaActionTypes {
+  updateSidekick = 'layout/saga/updateSidekick',
 }
 
-export interface LayoutState {
-  value: AppLayout;
-}
+export const updateSidekick = createAction<UpdateSidekickPayload>(SagaActionTypes.updateSidekick);
 
 const initialState: LayoutState = {
   value: {
     isContextPanelOpen: false,
-    isSidekickOpen: true,
+    isSidekickOpen: false,
```

Comment:
We could maybe read from localStorage here to set the correct initial state/default?

Reply:
Something like?

```
function resolveFromLocalStorage(key) {
  const value = localStorage.getItem(key);

  if (value === null || value === 'true') return true;

  return false;
}

const initialState: LayoutState = {
  value: {
    isContextPanelOpen: false,
    isSidekickOpen: resolveFromLocalStorage(SIDEKICK_OPEN_STORAGE),
    hasContextPanel: false,
  },
};

```